packages/integrations/vercel/test/fixtures/serverless-prerender/astro.config.mjs

```diff
@@ -2,6 +2,9 @@ import { defineConfig } from 'astro/config';
 import vercel from '@astrojs/vercel/serverless';
 
 export default defineConfig({
-	adapter: vercel(),
+	adapter: vercel({
+		// Pass some value to make sure it doesn't error out
+		includeFiles: ['package.json']
+	}),
```

Comment:
Shouldn't we assert that this file is copied over?

Reply:
I guess we could, but it won't be testing what the original issue had encountered.

Reply:
Added a test for it.